deployment/k8s/aws/aws-bootstrap.sh

```diff
@@ -19,7 +19,7 @@ branch="main"
 tag=""
 skip_git_pull=false
 
-base_dir="./opensearch-migrations"
+base_dir="../../../../opensearch-migrations"
```

Comment:
How did this work before?  Should this script switch/be relative to to the directory of the script?
```
  SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
  cd "${SCRIPT_DIR}"
```

Also - would it work if the repo was in a directory named something other than "opensearch-migrations"